frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -137,68 +182,125 @@ class PopupCreateFile extends React.Component {
         switch (xhr.status) {
           case 200:
             const jsonResult200 = JSON.parse(xhr.responseText)
-            this.handleClose()
-
-            GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.REFRESH_CONTENT_LIST, data: {} })
-
-            GLOBAL_dispatchEvent({
-              type: CUSTOM_EVENT.OPEN_CONTENT_URL,
-              data: {
-                workspaceId: jsonResult200.workspace_id,
-                contentType: state.appName,
-                contentId: jsonResult200.content_id
-              }
-            })
+
+            filePosted.jsonResult = { ...jsonResult200, code: 200 }
+
+            uploadedFileUpdatedList.push(filePosted)
+            this.setState({ uploadedFileList: uploadedFileUpdatedList })
             break
           case 400:
             const jsonResult400 = JSON.parse(xhr.responseText)
+
+            let errorMessage = props.t('Error while creating file')
             switch (jsonResult400.code) {
-              case 3002: this.sendGlobalFlashMessage(props.t('A content with the same name already exists')); break
-              case 6002: this.sendGlobalFlashMessage(props.t('The file is larger than the maximum file size allowed')); break
-              case 6003: this.sendGlobalFlashMessage(props.t('Error, the shared space exceed its maximum size')); break
-              case 6004: this.sendGlobalFlashMessage(props.t('You have reach your storage limit, you cannot add new files')); break
+              case 3002: errorMessage = props.t('A content with the same name already exists'); break
+              case 6002: errorMessage = props.t('The file is larger than the maximum file size allowed'); break
+              case 6003: errorMessage = props.t('Error, the shared space exceed its maximum size'); break
+              case 6004: errorMessage = props.t('You have reach your storage limit, you cannot add new files'); break
             }
+            filePosted.jsonResult = jsonResult400
+            filePosted.errorMessage = errorMessage
+            uploadedFileUpdatedList.push(filePosted)
+            this.setState({ uploadedFileList: uploadedFileUpdatedList })
+            break
+          default:
+            filePosted.jsonResult = { code: 0 }
+            filePosted.errorMessage = props.t('Error while creating file')
+            uploadedFileUpdatedList.push(filePosted)
+            this.setState({ uploadedFileList: uploadedFileUpdatedList })
             break
-          default: this.sendGlobalFlashMessage(props.t('Error while creating file')); break
         }
       }
     }
 
     xhr.send(formData)
   }
 
+  handleValidate = async () => {
+    const { state } = this
+
+    this.setState({ uploadStarted: true })
+
+    state.uploadFileList.forEach((file, index) => this.postFile(file, index))
+  }
+
+  handleDeleteFile = (file) => {
+    const { state } = this
+
+    const uploadFileWithoutDeletedFileList = state.uploadFileList.filter(f => f !== file)
```

Comment:
don't compare objects. Objects strict equality is a complicated thing.
You need another way to identify your files